doc/io.rst

```diff
@@ -338,6 +338,34 @@ supported by netCDF4-python: 'standard', 'gregorian', 'proleptic_gregorian' 'nol
 By default, xarray uses the 'proleptic_gregorian' calendar and units of the smallest time
 difference between values, with a reference time of the first time value.
 
+.. _io.iris:
+
+Iris
+----
+
+The Iris_ tool allows easy reading of common meteorological and climate model formats
+(including GRIB and UK MetOffice PP files) into ``Cube`` objects which are in many ways very
+similar to to ``DataArray``s, while enforcing a CF-compliant data model. If iris is
+installed xarray can convert a ``Cube`` into a ``DataArray`` using
+:py:meth:`~xarray.Dataset.from_iris`:
+
+.. ipython:: python
+
+    da_cube = xr.Dataset.from_iris(cube)
+    da_cube
+
```

Comment:
Are these examples working? If you'd like this code to run at build time you'll have to add iris to the doc build environment and define the ``cube`` variable first.

Alternatively, you can use the ``:verbatim:`` environment (see OPeNDAP example below) which is static and therefore easier (and more error prone on the long term, though)